numba_dpex/tests/kernel_tests/test_arg_types.py

```diff
@@ -37,13 +40,39 @@ def input_arrays(request):
 
 @pytest.mark.parametrize("filter_str", filter_strings)
 def test_kernel_arg_types(filter_str, input_arrays):
-    kernel = dpex.kernel(mul_kernel)
-    a, actual, c = input_arrays
+    usm_type = "device"
+
+    a, b, c = input_arrays
     expected = a * c
+
     device = dpctl.SyclDevice(filter_str)
```

Comment:
We want to get away from this pattern as it leads to lots of skipped tests when the filter string is not supported. Instead of using the filter string parameter, just set the device to `dpctl.select_default_device()`. In future, we will remove all filter string params and control default device using SYCL environment variables.